manifests/application.pp

```diff
@@ -24,11 +24,14 @@
   include certs::candlepin
   include certs::foreman
   include certs::pulp_client
-  include certs::qpid
   include katello::params
 
-  include katello::qpid_client
-  User<|title == $foreman::user|>{groups +> 'qpidd'}
+
+  if $facts['os']['release']['major'] == '7' {
+    include certs::qpid
+    include katello::qpid_client
+    User<|title == $foreman::user|>{groups +> 'qpidd'}
+  }
```

Comment:
Does katello still connect to qpid? Can't we drop this altogether or move it to `pulp.pp`?

Reply:
To keep tracking: https://github.com/theforeman/puppet-katello/pull/336

Reply:
I've included these updates in #336 so this PR now depends on those changes to go forward.